project_templates/roundtable_aiohttp_bot/{{cookiecutter.repo_name}}/manifests/base/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ cookiecutter.package_name }}
+  labels:
+    app: {{ cookiecutter.package_name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: {{ cookiecutter.package_name }}
+  template:
+    metadata:
+      labels:
+        name: {{ cookiecutter.package_name }}
+    spec:
+      containers:
+        - name: app
+          imagePullPolicy: "Always"
+          image: "lsstsqre/{{ cookiecutter.package_name }}:latest"
```

Comment:
This should be a version tag. `:latest` will mean Argo CD will upgrade the app as soon as a new Docker image is pushed, which we don't want. We need some release process that automatically updates the version tag, but in the interim I think it still needs to be a manual release step.

Reply:
I agree. This is something that a developer/operator needs to update, and having some tooling to update tags in conjunction with development and deployment would be awesome.

To try to address part of the issue, I've now left the tag off the `image` field in the Deployment and added an `image` field to the `kustomization.yaml`:

```yaml
images:
  - name: lsstsqre/{{ cookiecutter.package_name }}
    newTag: 0.0.0
```

So a developer could update `newTag` in the application's `kustomization.yaml`, or an operator could set `newTag` in the `kustomization.yaml` in the Roundtable repository.